sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -90,6 +93,8 @@ export class Receiver {
     this._context = context;
     this._partitionId = partitionId;
     this._receiverOptions = options || {};
+    this._streamingReceiver = {} as StreamingReceiver;
+    this._batchingReceiver = {} as BatchingReceiver;
```

Comment:
Why do we want to initialize the receivers this way?

Reply:
Updated!!